apps/learninglocker/templates/services/app/_dc_base.yml.j2

```
@@ -41,10 +54,26 @@ spec:
           env:
             - name: MONGODB_PATH
               value: "{{ mongodb_uri(env_type, trashable_env_types, learninglocker_mongodb_host, deployment_stamp, learninglocker_endpoint_mongodb_ips, learninglocker_mongodb_replicaset, learninglocker_mongodb_read_preference) }}"
+            # PM2 creates files and directories in the user's HOME. By default the user's HOME is `/`
+            # and obviously it's not possible for an unprivileged user to write to the `/` directory.
+            # PM2 will work as expected after changing the `HOME` environment variable to a
+            # directory the user can write to.
+            - name: HOME
+              value: /home/pm2_user
           volumeMounts:
             - name: learninglocker-v-storage
               mountPath: /app/storage
+            - name: learninglocker-configmap
+              mountPath: /pm2_config
```

The following is more UNIX-friendly IMO.

```suggestion
              mountPath: /usr/local/etc/pm2/config.d
```